### PR TITLE
Separate chain registration from`createNewChain` method

### PR DESCRIPTION
**Pull request by Raid5594**
…f choice

# What ❔

## Checklist

- [ ] PR title corresponds to the body of PR (we generate changelog entries from PRs).
- [ ] Tests for the changes have been added / updated.
- [ ] Documentation comments have been added / updated.

### CODE REVIEW

File: l1-contracts/contracts/bridgehub/Bridgehub.sol [hunk 1]
```diff
@@ -243,22 +274,14 @@ contract Bridgehub is IBridgehub, ReentrancyGuard, Ownable2StepUpgradeable, Paus
         bytes calldata _initData,
```
Comment (kelemeno) [inline l1-contracts/contracts/bridgehub/Bridgehub.sol:274]: you can just put assetId in this function rather than baseToken address

Comment (StanislavBreadless) [reply l1-contracts/contracts/bridgehub/Bridgehub.sol:274]: @kelemeno the problem with it is that in multiple places (such as diamond proxy storage) the base token is expected to be an address and not an id

**Comment by kelemeno** [comment]
Can we close this in favour of https://github.com/matter-labs/era-contracts/pull/700/

**Comment by kelemeno** [comment]
I am assuming yes, reopen if needed

**Pull request closed without merging**